app/enterprise/2.1.x/deployment/classic-deployment.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Classic Deployment
+---
+{{site.ee_product_name}} can be deployed in the following classic deployment modes:
+* [**Embedded:**](#embedded) classic deployment with all services on one node.
+* [**Distributed:**](#distributed) classic deployment with distributed services.
```

Comment:
```suggestion
* [**Embedded:**](#embedded) deployment with all services on one node.
* [**Distributed:**](#distributed) deployment with distributed services. Unlike 
in Hybrid mode, every node must connect to a database.
```
No need to repeat classic since it's in the intro line before the list. 

Also adding a comparison to Hybrid to differentiate - since there are no side-by-side images anymore, nice to have a point of reference.